docs/_sidebar.md

```diff
@@ -6,6 +6,8 @@
     - [Arma](guides/arma.md)
     - [Formacje](guides/formations.md)
     - [Robienie Misji](guides/missionmaking.md)
+    - [Symlink - ArmA SSD, mody HDD](https://drive.google.com/file/d/1lWbsHtAd6JXc7X-ZiMUpWLUAMQwwKrw9/view)
+    - [Wzywanie artylerii](https://drive.google.com/file/d/17DtQ8Z-7kXtwKS9dSXE29iuRrXvg9yse/view)
```

Comment:
It would be better if pages were created for this and links were available there. That will make it easier to organize things later and would not confuse user when he does not expect redirection.

Reply:
New pages created, with some chit-chat introduction, along with the links to outside sources.